### PR TITLE
Add a --browse option in CLI

### LINKED ISSUE #465: UX / CLI: add a --browse option
### Description the feature
Add a `--browse` /  `-b` option to the CLI to open the report in the browser instead of just printing where it is, to improve UX.

### Do you have any suggestions on how to add this feature in scanapi ?
Add the option, and in an `if` put a `webbrowser.open_new_tab` call below the existing logging call.

### PR DESCRIPTION
**Pull request by renato04**
## Description
I added a new option to open the report result on a browser.

## Motivation behind this PR?
Closes #465

## What type of change is this?

It is a new feature

## Checklist
 

- [x]  I have added a changelog entry / my PR does not need a new changelog entry. [Instructions](https://github.com/scanapi/scanapi/wiki/Changelog).
- [x] I have added/updated unit tests. [Instructions](https://github.com/scanapi/scanapi/wiki/Writing-Tests).
- [x] New and existing unit tests pass locally with my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests)
- [x] I have self-documented code my changes by adding docstring(s) and comment(s). [Instructions](https://github.com/scanapi/scanapi/wiki/First-Pull-Request#7-make-your-changes)
- [ ] Current PR does not significantly decrease the code coverage and docstring coverage.
- [x] My code follows the style guidelines of this project.
- [x] I have run ScanAPI locally and manually tested my changes. [Instructions](https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally).
- [x] I have squashed my commits. [Instructions](https://github.com/scanapi/scanapi/wiki/Squashing-Commits).

## Issue

Closes #465

### CODE REVIEW

File: scanapi/scan.py [hunk 1]
```diff
@@ -69,6 +72,13 @@ def write_report(results):
     reporter.write(results)
 
 
+def open_report_in_browser():
+    """Open the results file on a browser
```
Comment (Pradhvan) [inline scanapi/scan.py:76]: @renato04 if it's one line doc string it's good if the closing quotes are ends on the same line. That way static analysis won't fail too. 😄

File: scanapi/settings.py [hunk 2]
```diff
@@ -23,6 +23,7 @@ def __init__(self):
         self["output_path"] = None
         self["template"] = None
         self["no_report"] = False
+        self["open_browser"] = False
 
         super(dict, self).__init__()
```
Comment (Pradhvan) [inline scanapi/settings.py:28]: Can you update the super call? From `super(dict, self).__init__()`  -> `super().__init__()` that way I guess static analysis CI would pass.

**Review by Pradhvan — CHANGES_REQUESTED** [review]

**Comment by Pradhvan** [comment]
@renato04 great job ⭐  Just left some minor comments for the static analysis to pass.

**Comment by renato04** [comment]
Thanks @Pradhvan I'll fix it as soon as possible

**Comment by renato04** [comment]
@Pradhvan I fixed it!

**Review by Pradhvan — APPROVED** [review]
LGTM 👍🏾  @camilamaia what do you think ?

**Comment by hebertjulio** [comment]
Look good to me. Browser open with successful! :rocket: :smile:

**Comment by Pradhvan** [comment]
> Look good to me. Browser open with successful! rocket smile

Great! Thanks @hebertjulio  for reviewing it. Feel free to approve the PR. 😄

**Review by hebertjulio — APPROVED** [review]

**Pull request merged**